GROWTH_PLAN_TEMPLATE.md

```diff
@@ -21,7 +21,7 @@ So making sure that all Growth requirements are met is a good first step in any
 | --- | ---| --- | --- | --- | --- | 
 | Governing body of >= 5 members of which no more than 1/3 is affiliated with the same employer. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
 | Publicly documented governance, decision-making, contribution, and release processes. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
-| Healthy number of committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
+| Healthy number of regular committers from at least two organizations. | {CURRENT_STATUS} | {DESIRED_STATUS} | {PLAN} | {DATE} | <!--✅--> |
```

Comment:
why organizations? What about unaffiliated individuals?

Reply:
That's a fair question. Same answer as for https://github.com/openjs-foundation/project-status/pull/56#issuecomment-736691371, though: I don't want us to move too much away from [`PROJECT_PROGRESSION.md` expectations](https://github.com/openjs-foundation/cross-project-council/blob/master/PROJECT_PROGRESSION.md#iii-stages---definitions--expectations).

Would you and @christian-bromann want to open issues in the CPC repo about these?